procrastinate/aiopg_connector.py

```diff
@@ -1,26 +1,70 @@
-from typing import Any, Awaitable, Dict, Optional
+import asyncio
+from typing import Any, Awaitable, Dict, List, Optional
 
 import aiopg
-from psycopg2.extras import RealDictCursor
+import psycopg2.sql
+from psycopg2.extras import Json, RealDictCursor
 
-from procrastinate import psycopg2_connector, store
+from procrastinate import store
 
 
-def get_connection(**kwargs) -> Awaitable[aiopg.Connection]:
-    return aiopg.connect(**kwargs)
+def wrap_json(arguments: Dict[str, Any]):
+    return {
+        key: Json(value) if isinstance(value, dict) else value
+        for key, value in arguments.items()
+    }
```

Comment:
Why not registering an adapter for dict instead?

Reply:
Because I had kinda forgotten that psycopg2's adapters were... A thing?

Thankfully, I think there's a reference implementation lying around somewhere :sweat_smile: (like in the codebase of my main project that I shamelessly stole from yours ;) )

Reply:
I suggest we let this be done in #99